modules/aggregation-layers/src/contour-layer/marching-squares-codes.js

```diff
@@ -149,6 +157,37 @@ export const ISOLINES_CODE_OFFSET_MAP = {
   15: []
 };
 
+export const LINEAR_INTERPOLATION_CODE_OFFSET_MAP = {
+  // key is equal to the code of 4 vertices (invert the code specified in wiki)
+  // value can be an array or an Object
+  // Array defines the cell corners (top, current, topRight, right)
+  // Object : to handle saddle cases, whose output depends on mean value of all 4 corners
+  //  key: code of mean value (0 or 1)
+  //  value: Array , as above defines
+
+  0: [],
```

Comment:
This table has same format as `ISOLINES_CODE_OFFSET_MAP`, reduce the duplication and potential errors, this can be converted into a function, which takes either `OFFSET` or `CORNERS` has an argument.